manifests/s/SamyGO/PVRDecoder/1.0/SamyGO.PVRDecoder.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: SamyGO.PVRDecoder

```

```suggestion
PackageIdentifier: SamyGo.PVRDecoder
```
Must match existing publisher casing or there will be Git conflicts

---

SamyGO is correct name.

---

An Internet search shows both letters of GO are capitalized. The other manifests will need to be updated to match this one.

---

> An Internet search shows both letters of GO are capitalized. The other manifests will need to be updated to match this one.

In that case, the old manifests will need to be *completely removed*, wait for the publish pipeline to publish all changes removing the SamyGo manifests, then they will need to all be added back individually with the new SamyGO name.

Doing it any other way will result in directory conflicts

---

Pending https://github.com/microsoft/winget-pkgs/issues/111198